src/posix/platform/netif.cpp

```diff
@@ -469,6 +1201,7 @@ static void processMLDEvent(otInstance *aInstance)
     size_t              offset;
     uint8_t             type;
     struct ifaddrs *    ifAddrs = NULL;
+    char                addressString[INET6_ADDRSTRLEN + 1];
```

Comment:
Is this variable used?

Reply:
Yes, in inet_ntop call for the log messages.